### PR TITLE
Add a basic Angular app in waiting-room-ui

### PR DESCRIPTION
**Pull request by Biao**
All files are generated by running this Angualr CLI: ng new ui
Land it first so it is more clear what changes are made by us.

### CODE REVIEW

**Comment by dzelemba** [comment]
Should we call the top level folder "frontend" or "ui" instead of "waiting-room-ui" to be more future proof?

Other than that looks fine, but I didn't take a very close look at the Angular specifics as I'm not very familiar there. Would be good to get someone who is familiar with Angular to take a look before merging.

**Comment by Biao** [comment]
> Should we call the top level folder "frontend" or "ui" instead of "waiting-room-ui" to be more future proof?
> 
> Other than that looks fine, but I didn't take a very close look at the Angular specifics as I'm not very familiar there. Would be good to get someone who is familiar with Angular to take a look before merging.

Changed the folder name to ui. I will add Andy once he finished setting up github account.

File: ui/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# WaitingRoomUi
```
Comment (klepack) [inline ui/README.md]: nit: WaitingRoomUi was the old name

**Review by klepack — APPROVED** [review]

File: ui/README.md (hunk 1 above)
Comment (Biao) [reply ui/README.md]: done

**Pull request merged**